plugins/spectrum_analyzer.c

```diff
@@ -398,11 +405,6 @@ static bool configure_data_capture(plugin_setup *setup)
 		dev_info->adc_scale = '?';
 		dev_info->adc_freq = 0.0;
```

Comment:
maybe also change this to `dev_info->adc_freq = 0;`
no idea if some compiler warnings could trigger

Reply:
oh; wait; `adc_freq` is still double

Reply:
`dev_info->adc_freq` remains of type float so no actions needed here